astropy/modeling/tests/test_core.py

```diff
@@ -380,3 +388,13 @@ def test_compound_deepcopy():
     assert id(model._submodels[0]) != id(new_model._submodels[0])
     assert id(model._submodels[1]) != id(new_model._submodels[1])
     assert id(model._submodels[2]) != id(new_model._submodels[2])
+
+
+@pytest.mark.skipif('not HAS_SCIPY')
+def test_units_with_bounding_box():
+    points = np.arange(10, 20)
+    table = np.arange(10) * u.Angstrom
+    t = models.Tabular1D(points, lookup_table=table)
+
+    assert isinstance(t(10), u.Quantity)
+    assert isinstance(t(10, with_bounding_box=True), u.Quantity)
```

Comment:
It might be good to also check the actual result value - what about using ``assert_allclose_quantity``?